pylint/checkers/typecheck.py

```diff
@@ -70,6 +70,7 @@
 
 import astroid
 from astroid import nodes
+from astroid.const import PY310_PLUS
```

Comment:
`PY310` should probably be imported from `pylint.const`

Reply:
```suggestion
from pylint.constants import PY310_PLUS
```